### PR TITLE
Change on-call rotation period and add members to on-call duty

### PR DESCRIPTION
**Pull request by mimotej**
This PR changes rotation period from `weekly` to `daily` .  It also adds members to on-call rotation

### CODE REVIEW

**Comment by durandom** [comment]
@mimotej let's also add the people who are on call

**Comment by mimotej** [comment]
> @mimotej let's also add the people who are on call

Added on-call members to list if somebody is missing or shouldn't be there let me know

File: slack-first/overlays/moc/smaug/config.yaml [hunk 1]
```diff
@@ -12,6 +10,30 @@ onCall:
...
+    - slack: U01UN55AYMC
+      github: codificat
+    - slack: U031AF44D8S
+      github: SamoKopecky
+    - slack: U03QZRNQ9K4
+      github: VannTen
+    - slack: U03HG0HAGBG
+      github: shreekarSS
+    - slack: U02C5HG86QP
+      github: KPostOffice
```
Comment (durandom) [inline slack-first/overlays/moc/smaug/config.yaml:36]: @mimotej please remove dystewart, goern, Gkrumbach07, codifcat, VannTen, shreekarSS, KPostOffice

**Review by durandom — CHANGES_REQUESTED** [review]

File: slack-first/overlays/moc/smaug/config.yaml [hunk 2]
```diff
@@ -1,17 +1,25 @@
...
+    - slack: U01S8JVDS65
+      github: durandom
+    - slack: U02CWCX960N
+      github: schwesig
+    - slack: U01S22XJD60
+      github: tumido
+    - slack: U0277G1VAAV
+      github: mimotej
+    - slack: U02HA8W357G
+      github: mayaCostantini
```
Comment (HumairAK) [inline slack-first/overlays/moc/smaug/config.yaml:20]: I do not see `mayaCostantini` on this list. And @VannTen (Max)  seems to be missing.

Comment (durandom) [reply slack-first/overlays/moc/smaug/config.yaml:20]: ACK

Comment (mimotej) [reply slack-first/overlays/moc/smaug/config.yaml]: For now I have removed maya and added max if there will be any other changes needed I will add them :+1:

Comment (durandom) [reply slack-first/overlays/moc/smaug/config.yaml]: @harshad16 we decided with @goern you should not be on call, but act as a thoth devops

**Comment by durandom** [comment]
@mimotej please remove @harshad16

**Comment by mimotej** [comment]
> @mimotej please remove @harshad16
Fixed :+1:

**Review by schwesig — COMMENTED** [review]
referring to the Slack chat:
@HumairAK: add max

**Review by durandom — COMMENTED** [review]
/approve
/lgtm

**Review by schwesig — APPROVED** [review]
/lgtm
/approve
slack & github ids checked

maybe sorting it alphabetically? by github?

**Pull request merged**